modules/storage-expanding-recovering-failure.adoc

```diff
@@ -7,13 +7,21 @@
 [id="expanding-recovering-from-failure_{context}"]
 = Recovering from failure when expanding volumes
 
-If expanding underlying storage fails, the {product-title} administrator can manually recover the persistent volume claim (PVC) state and cancel the resize requests. Otherwise, the resize requests are continuously retried by the controller.
+If a resize request fails or remains in a pending state, you can try again by entering another smaller resize value in `.spec.resources.requests.storage` for the persistent volume claim (PVC). The smaller value should be smaller than the one you tried that failed.
```

Comment:
So the wording is weird, specially the last part.

> The smaller value should be smaller than the one you tried that failed.

It should be rather, resize request can only be retried to a size *larger* than original size of the volume. {{product-title}} doesn't support shrinking of volumes and hence recovery from volume expansion failure is only possible to a value that is larger than original size.

Reply:
yes correct

Reply:
>My understanding is that if you try to expand a volume and it fails, you can try a value smaller than the value you tried to expand to that failed

If expansion failed for a size, then you can retry with any size smaller or larger than size for which expansion failed, as long as - size you specified is larger than original size.  Bottom line - size that was used for failed expansion has no bearing on size with which you can retry.